### PR TITLE
Include Teleport Connect reference in installation docs page

### PR DESCRIPTION
**Pull request by stevenGravy**

### CODE REVIEW

**Review by zmb3 — COMMENTED** [review]
Before we do this, we have to make sure that the supported versions here actually apply to Connect as well.

@ravicious can you review this for accuracy?

File: docs/pages/installation.mdx [hunk 1]
```diff
@@ -18,18 +18,21 @@ Teleport is officially supported on the platforms listed below. It is worth
...
 
-\[2] *Enhanced Session Recording requires Linux kernel v5.8+*.
+\[2] *`tsh` is a Command Line Client (CLI) and Teleport Connect is an Graphical User Interface (GUI) desktop client. See
+  [Using Teleport Connect](connect-your-client/teleport-connect.mdx) for usage and installation*.
 
-\[3] *Teleport server does not run on Windows yet, but `tsh` (the Teleport client)
+\[3] *Enhanced Session Recording requires Linux kernel v5.8+*.
+
+\[4] *Teleport server does not run on Windows yet, but `tsh` (the Teleport client)
 supports most features on Windows 10 and later.*
```
Comment (alexfornuto) [inline docs/pages/installation.mdx:36]: Should we mention Teleport Connect in this note?

**Review by alexfornuto — APPROVED** [review]
One suggestion, otherwise 👍 for copy.

File: docs/pages/installation.mdx (hunk 1 above)
Comment (stevenGravy) [reply docs/pages/installation.mdx]: added!

**Review by ravicious — APPROVED** [review]

File: docs/pages/installation.mdx [hunk 2]
```diff
@@ -18,18 +18,21 @@ Teleport is officially supported on the platforms listed below. It is worth
...
 running Teleport on UNIX variants other than Linux \[1].
 
-| Operating System | `teleport` Daemon | `tctl` Admin Tool | `tsh` User Client | Web UI (via the browser) | `tbot` Daemon |
+| Operating System | `teleport` Daemon | `tctl` Admin Tool | `tsh` and Teleport Connect User Clients [2] | Web UI (via the browser) | `tbot` Daemon |
 | - | - | - | - | - | - |
-| Linux v2.6.23+ (RHEL/CentOS 7+, Ubuntu 14.04+, and Debian 8+) \[2] | yes | yes | yes | yes | yes |
+| Linux v2.6.23+ (RHEL/CentOS 7+, Ubuntu 14.04+, and Debian 8+) \[3] | yes | yes | yes | yes | yes |
 | macOS v10.13+  (High Sierra)| yes | yes | yes | yes | yes |
-| Windows 10+ (rev. 1607) \[3] | no | no | yes | yes | no |
+| Windows 10+ (rev. 1607) \[4] | no | no | yes | yes | no |
```
[the remark this answers is not in the document]
Comment (ravicious) [reply docs/pages/installation.mdx:25]: Well, and the supported Windows version is more generous than ours. However, in [the release notes](
https://releases.electronjs.org/release/v22.0.0) for the newest stable version of Electron (v22), I found this:

> Electron will be ending support for Windows 7/8/8.1 after version 22.x.y [following Chromium's plan to end support](https://support.google.com/chrome/thread/185534985/sunsetting-support-for-windows-7-8-8-1-in-early-2023?hl=en). Older versions of Electron will continue to work, but no further updates will be made for these operating systems.

**Pull request merged**